frontend/src/component/project/Project/ProjectInsights/LeadTimeForChanges/LeadTimeForChanges.tsx

```diff
@@ -0,0 +1,261 @@
+import { Box, styled, Tooltip, Typography, useMediaQuery } from '@mui/material';
+import { useProjectDoraMetrics } from 'hooks/api/getters/useProjectDoraMetrics/useProjectDoraMetrics';
+import { useRequiredPathParam } from 'hooks/useRequiredPathParam';
+import { useMemo } from 'react';
+import { useTable, useGlobalFilter, useSortBy } from 'react-table';
+import {
+    Table,
+    SortableTableHeader,
+    TableBody,
+    TableCell,
+    TableRow,
+    TablePlaceholder,
+} from 'component/common/Table';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { Badge } from 'component/common/Badge/Badge';
+import { useConditionallyHiddenColumns } from 'hooks/useConditionallyHiddenColumns';
+import theme from 'themes/theme';
+
+const Container = styled(Box)(({ theme }) => ({
+    gridColumn: 'span 5',
+    backgroundColor: theme.palette.background.paper,
+    padding: theme.spacing(3),
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(2),
+    borderRadius: theme.shape.borderRadiusLarge,
+    overflowY: 'auto',
+    maxHeight: theme.spacing(100),
```

Comment:
I am not 100% sure yet, but adding scrollbar and setting max height to 800px, which is close to design.